### DISCUSSION TITLE
Numbers appended to filenames

### DISCUSSION THREAD

**Discussion by jimmybrigs** [slskd/slskd#1177] [Q&A] [answered]
I just gave SLSKD (docker version) a try and I really like it over the desktop version. The only thing I havent figured out is how do I stop it from appending random numbers to the filename? 

I have tried browsing around and not sure I've seen other people with the same issues and  might have missed the documentation somewhere, but for example files are saving as: "name_638577324656322268.ext" (the number is different each time). Any help is appreciated.

**Comment by jpdillingham** [comment]
It does this when there is a file with the same name in the destination directory already.  Are you finding that it does it all the time?

**Reply by jimmybrigs** [reply, marked as answer]
Yes it was doing it for all files but thanks to your response it got me thinking that I needed to change my config. 

When I setup the docker container I setup two volumes (one for configs and one for the files). I hadnt bothered to setup Incomplete's into a separate directory, which I'm assuming was making it treat each file as a duplicate and therefore appending the number.

By adding in a directory for incomplete's it resolved the issue. Thank you!
